app/assets/stylesheets/forever_style_guide/modules/_hero.scss

```diff
@@ -70,7 +74,7 @@ TODO: use picture element + picturefill to avoid css hacks
   background-repeat: no-repeat;
 
   @media (min-width: $screen-xs) {
-    padding-top: 34%; /* (484 / 1400 * 100) image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
+    padding-top: percentage($hero-img-height / $hero-img-width); /* image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
```

Comment:
I think it's useful to include the image sizes in there regardless, so there's a record of what size the images need to be to work with these styles.